modules/ROOT/pages/managing-apis.adoc

```diff
@@ -41,49 +40,54 @@ To add APIs to your portal:
 
 . From the *API management* page, select the *APIs in Exchange* tab.
 . To search for an API by name or description, enter the name or description into the *Search* bar.
-. Select an API from the list, click the more options icon, and select *Add to portal*.
+. Locate the API in the list, click *+ (Add to the portal)*.
```

Comment:
Does the UI really have "the" in the text? (ew)

Reply:
Dev just got around to changing this for the GA.